scripts/complete_liquidity_provision.js

```diff
@@ -161,11 +171,20 @@ module.exports = async (callback) => {
     )
 
     console.log("==> Sending the order placing transaction to gnosis-safe interface, please execute this transaction first")
-    const nonce = (await masterSafe.nonce()).toNumber()
-    await signAndSend(masterSafe, orderTransaction, argv.network, nonce)
+    let nonce = argv.nonce
+    if (nonce === undefined) {
+      nonce = (await masterSafe.nonce()).toNumber()
+    }
+    await signAndSend(masterSafe, orderTransaction, argv.network, nonce, dry_run)
 
     console.log("==> Sending the funds transferring transaction, please execute this transaction second")
-    await signAndSend(masterSafe, bundledFundingTransaction, argv.network, nonce + 1)
+    await signAndSend(masterSafe, bundledFundingTransaction, argv.network, nonce + 1, dry_run)
+
+    if (!dry_run) {
+      console.log(
+        `To verify the transactions run the same script with --dry-run --nonce ${nonce} --brackets ${bracketAddresses.join()}`
```

Comment:
Nitpick: I'd use the `--option=value` format since it's the one we (tend to) use in the readme. 
```suggestion
        `To verify the transactions run the same script with --dry-run --nonce=${nonce} --brackets=${bracketAddresses.join()}`
```

Reply:
I agree it has become the standard but I prefer without. Of course this is a matter of opinion and we should probably remain consistent.